ContactDetailsApi/V2/Factories/EntityFactory.cs

```diff
@@ -14,13 +14,10 @@ public static class EntityFactory
     {
         public static ContactDetails ToDomain(this ContactDetailsEntity databaseEntity)
         {
-            if (databaseEntity == null) return null;
-            if (databaseEntity.ContactInformation == null) return null;
```

Comment:
Do these two checks not need to be retained? At least the databaseEntity null check on line 17, in case the assignments down in lines 29+ throw null exceptions?

Reply:
I assumed this change was because of a comment on a previous PR, but I can't find it! I'll revert it.

Reply:
Ideally you might want to put a test around this to handle what happens if/when `ContactDetailsEntity` is null